erpnext/manufacturing/doctype/work_order/work_order.py

```diff
@@ -73,6 +73,9 @@ def onload(self):
 		self.set_onload("material_consumption", ms.material_consumption)
 		self.set_onload("backflush_raw_materials_based_on", ms.backflush_raw_materials_based_on)
 		self.set_onload("overproduction_percentage", ms.overproduction_percentage_for_work_order)
+		self.set_onload(
+			"overproduction_percentage_for_work_order", ms.overproduction_percentage_for_work_order
+		)
```

Comment:
Use overproduction_percentage field, why new field overproduction_percentage_for_work_order?

Reply:
just missed it